proposals/NNNN-New_rules_for_providing_VRHelpItems_VRHelpTitle.md

```diff
@@ -0,0 +1,79 @@
+# Handling VR help requests when application does not send VR help prompt
+
+* Proposal: [SDL-NNNN](NNNN-New_rules_for_providing_VRHelpItems_VRHelpTitle.md)
+* Author: [Irina Getmanets](https://github.com/GetmanetsIrina)
+* Status: **Awaiting review**
+* Impacted Platforms: Core, RPC
+
+## Introduction
+
+"vrHelp" items are commands in Voice Recognition(VR) menu, "helpPrompt" is the message to speak by HMI that initiated by voice 'Help' command.
+"vrHelp" items and "helpPrompt" are set by [SetGlobalProperties](https://github.com/smartdevicelink/sdl_core/blob/master/src/components/interfaces/MOBILE_API.xml#L3125) for helping user to make voice navigation over registered application, what is very useful in case your hands are busy with driving.
+Safety requirements are very strong in automotive, so possibility to use Voice Recognition should be provided for driver all the time.
+
+## Motivation
+
+If there is no SetGlobalProperties issued, the help items list will be populated with command list sent to voice module or with default values from [smartDeviceLink.ini](https://github.com/smartdevicelink/sdl_core/blob/master/src/appMain/smartDeviceLink.ini#L122) file. If application issues a SetGlobalProperties with help item list, then this list will have precedence and it will be displayed on HMI.
+But there is no guarantee that Application provide "vrHelp" and "helpPrompt", and possible situation when driver couldn't use Voice Recognition.
```

Comment:
@GetmanetsIrina 
- odd comma after "helpPrompt"
- better to rephrase " .. and it is possible that driver won't be able to use .."